src/views/manage-posts/components/ask-ai.tsx

```diff
@@ -1,12 +1,9 @@
-import {
-  NButton,
-  NForm,
-  NFormItem,
-  NInput,
-  NPopover,
-  useMessage,
-} from 'naive-ui'
-import { OpenAI } from 'openai'
+import { NButton, NForm, NFormItem, NInput, NPopover, useMessage } from 'naive-ui';
```

Comment:
Don't break the code style. Plz use perttier format

Reply:
oh, I fixed it.